test/integration/scheduler/podsready/suite_test.go

```diff
@@ -59,8 +59,9 @@ func TestSchedulerWithWaitForPodsReady(t *testing.T) {
 func managerAndSchedulerSetupWithTimeout(mgr manager.Manager, ctx context.Context, value time.Duration) {
```

Comment:
Missing integration test for the new behavior

Reply:
Thanks, I will add test later